src/modules/control_allocator/ControlAllocation/ControlAllocationPseudoInverse.cpp

```diff
@@ -120,6 +120,8 @@ ControlAllocationPseudoInverse::allocate()
 	//Compute new gains if needed
 	updatePseudoInverse();
 
+	_prev_actuator_sp = _actuator_sp;
+
 	// Allocate
-	_actuator_sp = _actuator_trim + _mix * (_control_sp - _control_trim);
```

Comment:
@jlecoeur what was the original intent for `_control_trim`? My use here is to add an offset to an actuator, but the `_control_trim` logic leads to undesired effects, and I did not just want to remove it w/o knowing the background.

Reply:
Linearization around a specific actuator state. In a linear system the equation above reduces and trim could be removed. However in a non-linear case --for example a tiltrotor-- it is required.
What are the undesired effects?

Reply:
Makes sense. For example I added a trim of 0.2 to the right aileron for a FW setup, that resulted in a roll=pitch=0.2 control trim, which then overall resulted in an aileron setpoint of -0.2410.
Sounds like we need both then, as it's 2 different use-cases.